pkg/oauthserver/authenticator/interfaces.go

```diff
@@ -1,15 +1,14 @@
 package authenticator
 
 import (
-	"k8s.io/apiserver/pkg/authentication/user"
-
 	"github.com/openshift/origin/pkg/oauthserver/api"
+	"k8s.io/apiserver/pkg/authentication/authenticator"
 )
 
 type Assertion interface {
-	AuthenticateAssertion(assertionType, data string) (user.Info, bool, error)
+	AuthenticateAssertion(assertionType, data string) (*authenticator.Response, bool, error)
```

Comment:
both of these should take a context now I suppose

Reply:
> both of these should take a context now I suppose

They can, but you'll want to consider that separately.